.github/workflows/e2e-tests.yml

```
@@ -169,31 +161,24 @@
-        uses: useblacksmith/stickydisk@v1
-        with:
-          key: ${{ github.repository }}-${{ steps.node-version.outputs.version }}-npm-cache-${{ matrix.test-suite }}
-          path: ~/.npm
-      - name: Mount node_modules
-        uses: useblacksmith/stickydisk@v1
+          run_install: false
+      - uses: useblacksmith/setup-node@v5
         with:
-          key: ${{ github.repository }}-${{ steps.node-version.outputs.version }}-node-modules-${{ matrix.test-suite }}
-          path: ./node_modules
+          node-version: ${{ matrix.node-version }}
```

actionlint (1.7.7)

property "node-version" is not defined in object type {test-suite: any}

(expression)